jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue

```diff
@@ -14,18 +14,29 @@
       hint="Select the data to see metadata."
     />
 
+    <v-row v-if="has_primary">
+      <v-switch
+        label="Show primary header"
+        hint="Show MEF primary header metadata instead."
+        v-model="show_primary"
+        persistent-hint>
+      </v-switch>
+    </v-row>

```

Comment:
I'm not sure this is the most-intuitive.... but that can be covered in docs or iterated based on user-feedback (I don't really have any suggestions except a full extension-dropdown, but that might not be any more clear).